docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -0,0 +1,405 @@
+  files are listed, please correct or request for help.
+
+  * If you by mistake have not started with an updated version of the develop
+    branch you can get extra files listed.
+
+* If you want to correct something now before the pull request has been created,
+  do that in your local clone and do a new commit and push that to your fork.
+
+* Press "create pull request", write a nice description and press "create"
+  again.
+
+* If you go back to your own computer and just keep the clone as it is, you
+  can easily update the pull request if needed with more changes to the same
+  PO file. When the pull request has been merged by the Zonemaster work group,
+  you can delete the local clone and on your Github fork you can remove the
+  branch. Or keep them for next time.
+
+
+## Updates to msgids
+
+The *msgid* cannot be updated in the translation process. If you find an *msgid*
+that needs an update you can provide that information to the project. The
+preferred way is to create an issue in the relevant repository.
+
+* Message *(msgid)* in Zonemaster-Engine: [issue][new issue zonemaster-engine]
+* Message *(msgid)* in Zonemaster-CLI: [issue][new issue zonemaster-cli]
+* Message *(msgid)* in Zonemaster-Backend: [issue][new issue zonemaster-backend]
+
+If the message is in Zonemaster-Engine then include the message tag found in the
+header, e.g. "BASIC:NO_PARENT", in the issue decription. Also include the *msgid*
+as it is now and a suggestion for new language.
+
+
+## Adding a new language
+
+If you want to add a new language, then follow the translation steps above with
+some modifications. Before you add a language contact the Zonemaster project
+to discuss timeplan and other aspects of the new language. Every language should
+be updated at every new release of Zonemaster if there are updated or added
+messages to be translated.
+
+* First, as in the translation steps, create the branch to work in.
+  ```
+  git checkout origin/develop
+  git checkout -b translation-update
+  ```
+
+There is no PO file for new language, and it has to be created. The easiest way
+is to make a copy of an existing file.
+
+[ISO 639-1]: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
+[Wikipedia]: https://www.wikipedia.org/
+
+* Determine what language code to use for the new language should be. It should
+  be the correct [ISO 639-1]. By looking up the language on [Wikipedia] the
+  correct code can be determined.
+
+* I must be a code that is available in the *locale* system in the OSs that
+  Zonemaster supports. Try the following commands to see if it is available.
+  Replace "xx" with that code that you think it should be. Consult with
+  Zonemaster Project if in doubt.
+  ```
+  locale -a | grep xx_      # Works in FreeBSD and Rocky Linux
+  grep xx_ /etc/locale.gen  # Works in Debian and Ubuntu
+  ```
+
+* Go to the *share* and use an existing PO file, say sv.po, and make a copy
```

Comment:
`the *share*` -> `the *share* directory` - "The share" sounds like Dropbox or something. 
Maybe `, say sv.po,` -> `(e.g. sv.po)`?

Reply:
"directory" was missing. Not intended. Fixed. -- "say" also works.

Reply:
Yes, it works, but it's harder to parse.